NetFixClient.sln

```diff
@@ -7,6 +7,10 @@ Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "NetFixClient", "NetFixClien
 EndProject
 Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "OP2Internal", "OP2Internal\OP2Internal.vcxproj", "{97594BB1-793F-379B-F6AB-040AC8FE1424}"
 EndProject
+Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "op2ext", "..\op2ext\op2ext.vcxproj", "{6DE3610C-C9EE-4FD3-BBD7-66382C16FDE9}"
```

Comment:
This path doesn't look right. Why the "..\" here? That would place the op2ext project above the level of the solution file, rather than below it.

I suspect you are referencing a different copy of op2ext that exists locally on your system, which is outside of the NetFixClient folder, which is why it compiles for you, but fails on the CI servers.